assets/scss/editor.scss

```diff
@@ -0,0 +1,36 @@
+/**
+ * Style for the Elementor customizer.
+ */
+
+.elementor-panel-menu-item-hello-settings-header {
+	margin-top: 15px;
+	border-radius: 6px 6px 0 0;
+}
+
+.elementor-panel-menu-item-theme-style-form-fields {
+	border-radius: 0 0 6px 6px;
+}
+
+.elementor-nerd-box {
```

Comment:
It shouldn't be applied for all `.elementor-nerd-box ` instances.

Reply:
Good spot, I've added `.hello-elementor` wrapper around this.